contents/handbook/growth/sales/contract-rules.md

```diff
@@ -146,6 +146,16 @@ All free credits associated with startup plan roll-offs are one-time only, and s
 
 For contracting purposes, these free credits should either be applied before the contract term or included in the 12 month credit amount. If they are being applied before the contract term, adjust the contract date to start 2 months later and the one-time credits can be applied to cover the 2 invoices before the contract start date.
 
+### Margin negative deals
+
+In exceptional circumstances, we may explore providing additional discounts which eat into our operating margin for the following cases:
+
+1. They are a strategic logo we'd like to land as a brand-new customer.
+2. We are taking their business from a competitor.
+3. We are preventing them from churning to a competitor.
+
+If you believe you have a customer who falls into one of these categories and would like to provide additional credit/discount then in the first instance run through the opportunity details with Simon. If he's happy that it's the right thing to do, then you also need to seek approval from Charles and Raquel.
```

Comment:
Just trying to think process-lite here - can we have one person just approve it? If someone gives me the opp and also can present the cost and what our actual margin will be, I'm happy to just be the solo point of approval.

Reply:
Cool I'll make it you then, Raquel still expressed interest in being involved so I'll keep her in.